app/steps/ui/thankyou/template.html

```diff
@@ -23,23 +23,31 @@ <h2 class="heading-medium">{{content.successHeading1 | safe}}</h2>
                 <p>{{content.successParagraph2 | safe}}</p>
                 <p>{{content.successParagraph3 | replace('{helpLineNumber}', helpline.number) | safe}}</p>
 
+                {% if fields.displaySaveApplication.value === 'true' %}
                 <div>
                     <h2 class="heading-medium">{{content.saveYourApplication}}</h2>
                     <p>{{content.saveParagraph1}}</p>
+                    {% if fields.checkAnswersSummary.value == 'true' %}
                     <p class="save-this-page-link">
                         <i class="icon icon-file-download"></i>
                         <a class="anchor-margin-left" id="checkAnswerHref" href="/check-answers-pdf">{{content.checkAnswersPdf}}</a>
                     </p>
+                    {% endif %}
+                    {% if fields.legalDeclaration.value === 'true' %}
                     <p>
                         <i class="icon icon-file-download"></i>
                         <a class="anchor-margin-left" id="declarationPdfHref" href="/declaration-pdf">{{content.declarationPdf}}</a>
                     </p>
+                    {% endif %}
+                    <div class="margin-top50 panel panel-border-wide">
+                        <p>{{content.successParagraph4 | replace('{findOutNext}', links.findOutNext) | safe}}</p>
+                    </div>
```

Comment:
Rather than duplicating this content so it can have `margin-top50` in the wrapping `<div>`, it would be better to display this outside the conditional without the `margin-top50` and then put bottom margin on the `<div>` wrapping the download links